requirements.txt

```diff
@@ -18,3 +18,4 @@ sphinx>=4.4.0
 sphinx-rtd-theme>=1.0.0
 myst-parser>=0.17.0
 twine>=3.8.0
+dohq-artifactory==0.8.1
```

Comment:
It looks like you're missing `seeq-spy`. I tried to build from a clean venv and got:
```
⏚ [bryan:~/dev/seeq-plot-curve] [.venv] [ruby-2.5.1] task/kcbl/package-plot-curve-for-addon-manager-CRAB-39547 ± python package.py 
Traceback (most recent call last):
  File "/Users/bryan/dev/seeq-plot-curve/package.py", line 8, in <module>
    from seeq.addons.plot_curve import __version__
  File "/Users/bryan/dev/seeq-plot-curve/seeq/addons/plot_curve/__init__.py", line 1, in <module>
    from ._add_on_main import PlotCurve
  File "/Users/bryan/dev/seeq-plot-curve/seeq/addons/plot_curve/_add_on_main.py", line 1, in <module>
    from seeq.addons.plot_curve.ui_components import AppLayout
  File "/Users/bryan/dev/seeq-plot-curve/seeq/addons/plot_curve/ui_components/__init__.py", line 1, in <module>
    from ._chart_component import ChartComponent
  File "/Users/bryan/dev/seeq-plot-curve/seeq/addons/plot_curve/ui_components/_chart_component.py", line 9, in <module>
    from seeq.addons.plot_curve.backend import Equation
  File "/Users/bryan/dev/seeq-plot-curve/seeq/addons/plot_curve/backend/__init__.py", line 1, in <module>
    from ._equation import Equation
  File "/Users/bryan/dev/seeq-plot-curve/seeq/addons/plot_curve/backend/_equation.py", line 5, in <module>
    from seeq import spy
ImportError: cannot import name 'spy' from 'seeq' (unknown location)
```
It worked after I pip installed `seeq-spy`

Reply:
Fixed in the next push!